graph-db/serverless/src/utils/cmr/fetchPageFromCMR.js

```diff
@@ -19,16 +25,19 @@ export const fetchPageFromCMR = async (scrollId, token, gremlinConnection) => {
     requestHeaders['CMR-Scroll-Id'] = scrollId
   }
 
+  let fetchUrl = `${process.env.CMR_ROOT}/search/collections.umm_json?page_size=${process.env.PAGE_SIZE}&scroll=true`
+
+  if (providerId !== null) {
+    fetchUrl += `&provider=${providerId}`
+  }
+
   try {
-    const cmrCollections = await fetch(`${process.env.CMR_ROOT}/search/collections.umm_json?page_size=${process.env.PAGE_SIZE}&scroll=true`, {
+    const cmrCollections = await fetch(fetchUrl, {
       method: 'GET',
       headers: requestHeaders
     })
 
-    const { headers = {} } = cmrCollections;
-
-    // eslint-disable-next-line no-param-reassign
-    ({ 'CMR-Scroll-Id': scrollId } = headers)
+    const { 'cmr-scroll-id': cmrScrollId } = cmrCollections.headers.raw()
```

Comment:
What is `.raw()` doing?

Would also prefer destructuring used like it was previously instead of chaining `cmrCollections.headers`
```
const { headers = {} } = cmrCollections
const { 'cmr-scroll-id': cmrScrollId } = headers
```

Reply:
The code you suggested is what is on the baseline right now. It does not work. I have to call `cmrCollections.headers.raw()` to get the scroll id out.

Reply:
This is the reason that we were only getting the first page, then it stopped.